merkletree/pad.go

```diff
@@ -114,8 +114,9 @@ func (pad *PAD) GetSTR(epoch uint64) *SignedTreeRoot {
 func (pad *PAD) TB(key string, value []byte) (*TemporaryBinding, error) {
 	//FIXME: compute private index twice
 	//it would be refactored after merging VRF integration branch
-	index, _ := pad.computePrivateIndex(key)
-	tb := pad.currentSTR.sig
+	cStr := pad.currentSTR
```

Comment:
>>I think PAD needs a policies field reflecting the current policies in place. currentSTR may

>OK, I have to think about that (hopefully after the talks/program finished). But on first sight it makes sense to add a policies field to PAD.

Didn't have the time to think it through (yet). But do we still need the policies field in `STR` than (for serializing, or sth else)?